python/ql/src/experimental/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -397,7 +469,7 @@ class DataFlowType extends TDataFlowType {
 }
 
 /** A node that performs a type cast. */
-class CastNode extends Node {
+class CastNode extends CfgNode {
```

Comment:
I'm amused by this change. Is it setting up future work? :slightly_smiling_face:

Reply:
Heh, this must be a leftover from the beginnings of this PR.